### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by stevenwally**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+					<button class="btn btn-default" type="button">Go!</button>
+				</span>
+			</div><!-- /input-group -->
+		</div><!-- /.col-lg-6 -->
+	</div>
+</nav>
+<!-- MAIN CONTENT -->
+<div id="output" class="container-fluid"></div>
+<!-- BOOTSTRAP CDN -->
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```
Comment (gregkorte) [inline index.html]: Since bootstrap requires jQuery, lines 34 & 35 should be swapped.

File: scripts/JSONhandle.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+var Chatty = (function (oldChatty) {
```
Comment (gregkorte) [inline scripts/JSONhandle.js]: What's this file doing here? :)

File: scripts/eventHandlers.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+		}
+
+		function dynamicInput(e) {
+			for (var i = 0; i < inventory.length; i++) {
+					var newText = document.getElementsByClassName("card-change");
+				if (e.keyCode === 13) {
+					userInput.value = "";
+					carClick();
+				} else if (inventory[i].classList.contains("card-change")) {
+					newText[0].childNodes[4].innerHTML = userInput.value;
```
Comment (gregkorte) [inline scripts/eventHandlers.js:40]: Careful with this type of targeting. It makes for a fragile product. What happens if you switch where the description appears?

File: scripts/eventHandlers.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+
+		// DOM-RELATED EVENT LISTENERS
+		document.addEventListener("click", carClick);
+		userInput.addEventListener("keyup", dynamicInput);
+
+		// DOM-RELATED FUNCTIONS
+
+		function carClick () {
+			console.log(event.currentTarget);
```
Comment (gregkorte) [inline scripts/eventHandlers.js]: Your final code should contain no console logs.

**Comment by gregkorte** [comment]
Good work on your quiz, Steven! Check the few minor things I found, and make your corrections. Don't worry about the targeting, just keep it in mind. Let me know when you're ready for me to take another look.

File: scripts/JSONhandle.js (hunk 2 above)
Comment (stevenwally) [reply scripts/JSONhandle.js]: 😬 

...haha...whoopsies.

**Comment by gregkorte** [comment]
You are good to merge, Steven. :+1:

**Pull request merged**